src/components/App.js

```diff
@@ -72,18 +81,17 @@ const Nav = styled.nav`
   }
 `;
 
-
 const App = () => {
   // add logic here
   // - add metadata (e.g. title tag and description) > depending upon the active route
   // - Helmet module could be useful here ^
   return (
     <>
       <Router>
-
         <TopBar>
+        
           <TopContainer>
-            <Link to="/">[link to home - curenetics logo]</Link>
+          <Logo src={cureneticsLogo} alt="Curenetics Clinical Trials" />
```

Comment:
I think this should be a link which redirects us to the home page,
what do you think?

Reply:
@MohammedYehia amends made.